sys/src/cmd/cdfs/main.c

```diff
@@ -382,14 +383,14 @@ readctl(Req *r)
 	p = seprint(p, e, "maxspeed read %d write %d\n",
 		drive->maxreadspeed, drive->maxwritespeed);
 
-	if (drive->scsi.changetime != 0 && drive->ntrack != 0) { /* have disc? */
+	if (drive->changetime != 0 && drive->ntrack != 0) { /* have disc? */
```

Comment:
This looks like a bugfix - maybe would have been better to put this in separately?

Reply:
Derivated from deanon (abandoning anonymous structs in Plan9) that was done later than the first attempt to port cdfs. If you have a better solution... tell me.

Reply:
In this case, scsi was already a field in the Drive struct - it wasn't anonymous.  Both the Scsi and Drive structs have changetime fields.

Reply:
compiler warned. Let me back to original an see what's happening.